app/routes/_index/dashboard.tsx

```diff
@@ -102,21 +107,25 @@ export function Dashboard() {
 	);
 }
 
+function getTrendDirection(value: number): QuickStatProps['trendDirection'] {
+	if (value > 0) {
+		return 'positive';
+	}
+	if (value < 0) {
+		return 'negative';
+	}
+	return 'neutral';
+}
+
 interface QuickStatProps {
 	title: string;
 	stat: string;
 	trend: string;
 	icon: ReactNode;
-	trendDirection: 'positive' | 'negative';
+	trendDirection: 'positive' | 'negative' | 'neutral';
 }
 
-const QuickStat = ({
-	title,
-	stat,
-	trend,
-	icon,
-	trendDirection,
-}: QuickStatProps) => {
+const QuickStat = ({ title, stat, trend, trendDirection }: QuickStatProps) => {
```

Comment:
i think TS should be complaining about removing the `icon` prop and still using it above

Reply:
Went ahead and deleted the prop from the interface and from every usage of `<QuickStat />`